modules/cluster-dialer/auth/auth.go

```diff
@@ -88,11 +88,10 @@ func (a *Authorizer) Authorizer(req *http.Request) (string, bool, error) {
 	logrus.Debugf("get auth info: %s", authInfo)
 
 	// Doesn't need cache now
-	akSkResp, err := a.Credential.QueryAccessKeys(context.Background(), &credentialpb.QueryAccessKeysRequest{
-		AccessKey: authInfo,
-		Scope:     apistructs.CMPClusterScope,
-		ScopeId:   clusterKey,
-		Status:    credentialpb.StatusEnum_ACTIVATE,
```

Comment:
does `status` useless?

Reply:
yes, it is useless now. I kept the property in db for future usage.